Makefile

```diff
@@ -366,13 +366,13 @@ hack-test-%: ## Runs the specified script in ./hack/test with well known environ
 
 .PHONY: generate
 generate: ## Generates code from protobuf service definitions and machinery config.
-	@$(MAKE) local-$@ DEST=./ PLATFORM=$(OPERATING_SYSTEM)/$(ARCH) EMBED_TARGET=embed-abbrev
+	@$(MAKE) local-$@ DEST=./ PLATFORM=linux/$(ARCH) EMBED_TARGET=embed-abbrev
 
 .PHONY: docs
 docs: ## Generates the documentation for machine config, and talosctl.
 	@rm -rf docs/configuration/*
 	@rm -rf docs/talosctl/*
-	@$(MAKE) local-$@ DEST=./ PLATFORM=$(OPERATING_SYSTEM)/$(ARCH)
+	@$(MAKE) local-$@ DEST=./ PLATFORM=linux/$(ARCH)
```

Comment:
I think for docs it might even make sense to keep the arch `amd64` as changing it will result in overrides to defaults such as this one on each rerun on a different architecture.
https://github.com/siderolabs/talos/blob/bb90945344f02b9cdae6e0e01821792dca25096b/website/content/v1.2/reference/cli.md?plain=1#L95

Reply:
yes, I think that makes sense